### DISCUSSION TITLE
Training on CPU using deepxde, CPU core usage is too high

### DISCUSSION THREAD

**Discussion by p781709996** [lululxvi/deepxde#1798] [Q&A]
My server has 64 CPUs，When I use the deepxde(MIONET) for training on the CPU，I found that all 64 CPUs are all occupied，How to reduce CPU usage？
Thanks

**Comment by lululxvi** [comment]
This is a question for TensorFlow/PyTorch, rather than DeepXDE. But I don't know the answer.
